docs/build/training.md

```diff
@@ -41,6 +41,10 @@ redirect_from: /docs/how-to/build/training
 
    Refer to the [Android build instructions](./android.md) and add the ```--enable_training_apis``` build flag.
 
+   d. For MacOS
+
+   Refer to the [macOS inference build instructions](./inferencing.md) and add the `--enable_training_apis` build flag.
```

Comment:
should we add a section for iOS too?

Reply:
I am wondering where we should put the iOS documentation. It can either be here or [Build for iOS](https://onnxruntime.ai/docs/build/ios.html) section.

Reply:
I think we could have something similar to the "For Android" section above. just refer to the iOS build instructions and mention the `--enable_training_apis` flag

Reply:
add the instructions